projects/plugins/jetpack/extensions/blocks/contact-info/index.js

```diff
@@ -51,6 +51,17 @@ export const settings = {
 	supports: {
 		align: [ 'wide', 'full' ],
 		html: false,
+		color: {
+			link: true,
+			gradients: true,
```

Comment:
Note to self: It is implied that text & background color is enabled by default when the color object exists.